### ISSUE TITLE
fig0 revisions

### ISSUE THREAD

**Issue by wrgr** [openconnectome/ocp-journal-paper#56]
People seem to have opinions on how to make this better!

Discuss here.  :)

@jovo @alexbaden @gkiar @MrAE

**Comment by MrAE** [comment]
The newest version has a lot more going on.  It's not as simplistic as the first version.

**Comment by gkiar** [comment]
I've chatted a bit with @jovo about this fig. Personally, I find it a bit
cluttered and torn between two levels of content: It is in some ways very
low level, but in other ways quite superficial. I think as the initial
figure in our paper/that we turn to as our organization, we want to define
the primary message we wish to convey. If that message is along the lines
of "here is an ecosystem which ties together tools to help you store,
explore, process, and analyze your data", then the first one was much more
succinct. If the goal is a lower level description of "here are all the
pieces in the puzzle that do awesome things for you and here are some of
the ones we built with or on them that enable more awesome things" then
perhaps we go with the second, but clean it up to make that point more
clear.

tl;dr: **what is the take home message from this figure?**

## 

Greg Kiar gkiar07@gmail.com

**Comment by wrgr** [comment]
I love both figures.  The simple one is nice to get our point across as eye candy and I definitely see it for advertising.  But too simple to convey much actual info. The complicated one is good for detail, but we'd need to modify to tie in all the services and might be better for the website.  I vote for a hybrid.  Meaning high level but more detail than store explore analyze.  Or maybe a multipanel fig with both and some tweaks?

**Comment by jovo** [comment]
i recommend you guys look at some nature/nature methods papers that you
like, and check out their "fig0"
and report back on things that you like/dislike.

On Friday, June 10, 2016, William Gray notifications@github.com wrote:

> I love both figures. The simple one is nice to get our point across as eye
> candy and I definitely see it for advertising. But too simple to convey
> much actual info. The complicated one is good for detail, but we'd need to
> modify to tie in all the services and might be better for the website. I
> vote for a hybrid. Meaning high level but more detail than store explore
> analyze. Or maybe a multipanel fig with both and some tweaks?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/neurodata/ndpaper/issues/56#issuecomment-225216960,
> or mute the thread
> https://github.com/notifications/unsubscribe/AACjckqiRVDIK2OauI7UFNieh-BMMe2oks5qKYSmgaJpZM4IykoX
> .

## 

the glass is all full:  half water, half air.
neurodata.io, jovo calendar
https://calendar.google.com/calendar/embed?src=joshuav%40gmail.com&ctz=America/New_York
